sdk/nodejs/appsync/resolver.ts

```diff
@@ -70,23 +64,29 @@ import * as utilities from "../utilities";
  *     $utils.appendError($ctx.result.body, $ctx.result.statusCode)
  * #end
  * `,
- *     type: "Query",
+ *     cachingConfig: {
+ *         cachingKeys: [
+ *             `$context.identity.sub`,
```

Comment:
Is it intentional that we changed lots of string literals to use backticks?  Are we doing this when there is a literla `$` in the string now?  

cc @pgavlin @lblackstone

Reply:
Same answer re: the transition to TF12 syntax. I'm not 100% sure what's triggering this, but the `$`s are a good guess.